client/src/components/Game/index.js

```diff
@@ -36,7 +39,9 @@ class Game extends Component {
       .then((docSnapshot) => {
         if (docSnapshot.exists) {
           game.onSnapshot(snapshot => {
-          this.updateGame(snapshot.data());
+            this.updateGame(snapshot.data());
+            this.playerInGame(snapshot.data());
+            this.gameStarted(snapshot.data());
```

Comment:
nit: Update naming to be better. gameStarted _sounds_ like it should return a boolean of whether or not the game has started, while in reality it updates the gameStarted state. maybe `setGameStarted`? Also, what is the purpose of doing two extra methods instead of adding it to `updateGame`? As far as I can see, they're all only run when the game is updated and gets a new snapshot.

Reply:
i wasn't clear on how exactly that method worked, but you're right it would be better to move them.